### DISCUSSION TITLE
Feature Request: Ability to Darken Player Buffs/Debuffs

### DISCUSSION THREAD

**Discussion by kaminador** [SFX-WoW/Masque#161] [Issues] [answered]
### Game Flavor

Classic

### Game Version

1.14.1

### Add-On Version

9.1.5

### Describe the Improvement

Does Masque have the ability to apply darkening to player buffs/debuffs? I have looked through the options but don't see anything regarding this. Maybe I missed it! If not, the ability to darken player buffs/debuffs would be a lovely feature. Great addon overall - cheers.

### Screenshots

https://imgur.com/a/ieK4Uek

### Notes

The screenshot above is from someone's UI but I'm not sure how they achieved that look. Something like that would be ideal. Thanks for your time.

**Comment by StormFX** [comment]
Darkening/transparency is up to the buff/debuff add-on that's being used. Masque only provides the textures/colors.

**Comment by kaminador** [comment]
My apologies, I used the wrong adjective to describe what I was going for! I'm actually referring to the texture/color of the buff border, not making it darker/transparent. Sorry for the miscommunication.

**Comment by StormFX** [comment, marked as answer]
The buff border color is also controlled by the buff/debuff add-on.

**Comment by kaminador** [comment]
I see now. I guess what I'm looking for is a buff/debuff addon, s I'll continue the search elsewhere. Thank you for your time!
